### PR TITLE
Feature/sign page

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

New log in/trial pages
https://www.figma.com/file/RPLINi2nrD30hQjcANws3CxH/Sanbase?node-id=11515%3A65307

**Screenshots**
image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,11 +36,21 @@ import { mapSizesToProps } from './utils/withSizes'
 import styles from './App.module.scss'
 import './App.scss'
 
-const PATHS = {
-  FEED: '/feed'
+export const PATHS = {
+  FEED: '/feed',
+  LOGIN: '/login',
+  LOGIN_VIA_EMAIL: '/login/email',
+  CREATE_ACCOUNT_FREE_TRIAL: '/login/trial',
```
Comment (Partysun) [inline src/App.js]: @GarageInc /signup intead of trial for path. Cause actually this page is created for signup or create account action

File: src/App.js [hunk 2]
```diff
@@ -35,12 +35,23 @@ import LogoutPage from './pages/Logout/Logout'
...
-
-const PATHS = {
-  FEED: '/feed'
+import CreateAccountFreeTrial from './pages/Login/CreateAccountFreeTrial'
+
+export const PATHS = {
+  FEED: '/feed',
+  LOGIN: '/login',
+  LOGIN_VIA_EMAIL: '/login/email',
+  CREATE_ACCOUNT_FREE_TRIAL: '/signup',
```
Comment (haritonasty) [inline src/App.js]:
```suggestion
  CREATE_ACCOUNT_FREE_TRIAL: '/sign-up',
```

File: src/assets/login/gift.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...756 2.618.807.96 1.532 1.927 2.183 3.065.744 1.301.739 2.647.743 4.16-.03.926-.808 1.897-1.909 1.626-1.19-.35-2.28-.453-3.397-.976-1.12-.606-2.242-1.212-3.436-1.646-1.366-.508-2.746-1.267-3.808-2.297-1.86-1.823-1.948-4.76-.132-6.55z" fill="#E7EAF3"/><path d="M34.012 87.505a1 1 0 0 1 .163 1.57c-1.129 1.08-2.04 2.281-2.601 3.765a1 1 0 0 1-1.597.395c-1.304-1.15-2.557-2.09-4.125-2.838a1.001 1.001 0 0 1-.114-1.742c1.075-.696 2.286-1.873 2.978-3.837a1 1 0 0 1 1.66-.364c1.13 1.163 2.41 2.285 3.636 3.05zM46.23 73.653c-1.794-.411-3.542-1.076-5.292-1.65a.844.844 0 0 1-.559-.978c.299-1.412.534-2.853.457-4.338v-.001l-3.752-1.712a.846.846 0 0 1-.458-1.004c.314-1.09.516-2.188.673-3.283l.001-.005c.066-.442.162-.879.23-1.32-1.69-.632-3.366-1.301-5.054-1.937a.843.843 0 0 1-.542-.806c.036-1.873.172-3.575.51-5.267M176.051 89.26c-1.786.45-3.645.651-5.465.936a.845.845 0 0 1-.942-.617c-.377-1.393-.824-2.783-1.569-4.071h0l-4.12.18a.847.847 0 0 1-.865-.686c-.217-1.113-.537-2.183-.895-3.23l-.002-.004c-.142-.424-.255-.857-.397-1.28-1.792.205-3.589.371-5.381.572a.843.843 0 0 1-.85-.471c-.82-1.684-1.474-3.262-1.943-4.922" stroke="#2F354D" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
Comment (haritonasty) [inline src/assets/login/gift.svg]: How is it in night mode?

File: src/App.js (hunk 2 above)
Comment (haritonasty) [reply src/App.js]: We can use `CREATE_ACCOUNT` name, more general (if we decide delete trial feature in  future)

File: src/pages/Login/CreateAccountFreeTrial.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+              className={styles.icon}
+              width='17'
+              height='19'
+              viewBox='0 0 17 19'
+              fill='none'
+              xmlns='http://www.w3.org/2000/svg'
+            >
+              <path
+                d='M6.74023 9.73535C6.77366 9.75209 6.79036 9.76045 6.81543 9.76881C6.79036 9.76045 6.78201 9.75209 6.74023 9.73535Z'
+                fill='#181B2B'
```
Comment (haritonasty) [inline src/pages/Login/CreateAccountFreeTrial.js]: night mode - ok? I think, should be dynamic color

File: src/pages/Login/FreeTrialBlock.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import giftImg from './../../../src/assets/login/gift.svg'
+import Button from '@santiment-network/ui/Button'
+import { Link } from 'react-router-dom'
+import styles from './FreeTrialBlock.module.scss'
+import { PATHS } from '../../App'
```
Comment (haritonasty) [inline src/pages/Login/FreeTrialBlock.js]: rearrange imports

File: src/pages/Login/LoginEmailForm.js [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+        className={styles.emailInput}
+      />
+      <Button
+        variant='fill'
+        accent='positive'
+        className={styles.email__btn}
+        type='submit'
+        disabled={loading}
+      >
+        {loading ? 'Waiting...' : 'Continue'}
```
Comment (haritonasty) [inline src/pages/Login/LoginEmailForm.js]: We can use loading state for button

File: src/pages/Login/index.js [hunk 7]
```diff
@@ -2,28 +2,72 @@ import React from 'react'
...
 import styles from './index.module.scss'
 
-const baseLocation = '/login'
-
-const LoginOptions = () => (
-  <>
-    <h2 className={styles.title}>Log in your account with</h2>
+const LoginDescription = () => (
+  <div className={styles.loginBlock}>
+    <h3 className={styles.title}>Welcome to Santiment</h3>
```
Comment (haritonasty) [inline src/pages/Login/index.js]: @Partysun @GarageInc no?
```suggestion
    <h3 className={styles.title}>Welcome to Sanbase</h3>
```

File: src/pages/Login/index.js [hunk 8]
```diff
@@ -2,28 +2,72 @@ import React from 'react'
...
-        <Icon type='envelope' className={styles.btn__icon} />
-        Email
-      </Link>
+      <div className={styles.divider}>
+        <span className={styles.use}>or use</span>
+      </div>
+      <LoginEmailBtn />
+
+      <div className={styles.new}>
+        New to Santiment?{' '}
```
Comment (haritonasty) [inline src/pages/Login/index.js]:
```suggestion
        New to Sanbase?{' '}
```

File: src/assets/login/gift.svg (hunk 3 above)
Comment (GarageInc) [reply src/assets/login/gift.svg]: @haritonasty 
image

File: src/pages/Login/CreateAccountFreeTrial.js (hunk 4 above)
Comment (GarageInc) [reply src/pages/Login/CreateAccountFreeTrial.js]: @haritonasty 
image

File: src/assets/login/gift.svg (hunk 3 above)
Comment (haritonasty) [reply src/assets/login/gift.svg]: Let's code svg with vars

File: src/pages/Login/CreateAccountFreeTrial.js (hunk 4 above)
Comment (haritonasty) [reply src/pages/Login/CreateAccountFreeTrial.js]: yep, use color variable for bullets

**Pull request merged**